spec/factories/users.rb

```diff
@@ -1,7 +1,7 @@
 FactoryGirl.define do
   factory :user do
     email
-    full_name { Faker::Name.name }
+    full_name { Faker::Name.name.gsub("'", "") }
```

Comment:
@ruslankhaertdinov could it be done in the Faker in self? B/c/ there are more cases like `&`